test/tools/WebListener/NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="dotnet-core" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
+</configuration>
```

Comment:
Should we add this for TestExe?

Reply:
I only needed to add this as the latest aspnetcore isn't on nuget.org.  No changes to TestExe are needed.

Reply:
This `nuget.config` file doesn't introduce a new feed. Do we need it only because we need to try `dotnet-core` before `nuget.org`?

Reply:
Yes, to pick up newer version of aspnetcore than what is on nuget.org currently (otherwise it didn't work correctly on my macBook).  We should get rid of this in the future.